omp/__init__.py

```diff
@@ -7,10 +7,11 @@
 import sys
 
 try:
+    # there may be an environ modification when loading config
     from pythran import config
-    cxx = config.cfg.get('user', 'cxx')
 except ImportError:
-    cxx = 'c++'
+    pass
+cxx = os.environ.get('CXX', 'c++')
```

Comment:
Is it the correct way to do?
I mean. Here is a "hidden" behavior on os.environ through the import. But why not just using the os.environ["CXX"] variable?
I would say that it is done that way because different compiler may use differente libgomp version but it is false as each compiler use its own OpenMP Implementation. (Clang is using libiomp5.so for example).
Here, we only want a path to an openmp library. We don't care about what is the version use. We don't care if it is the clang one or the gcc one, we just want an implementation to call it (without threads) in the Python world.
IMO, we should just use the os.environ without Pythran modification.